docs/sources/administration/correlations/create-a-new-correlation/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Create a new correlation
+weight: 40
+---
+
+# Create a new correlation
+
+## Before you begin
+
+Make sure you have permission to add new correlations. Only users with write permissions to data sources can define new correlations.
+
+## Create a correlation in Administration page
+
+1. Go to the Administration section in Grafana
+1. Open Correlations page
+1. Click the “Add” button in the top right corner
+1. A wizard will guide you through the correlation setup
```

Comment:
why do we specify this if we describe the process anyway? doesn't sound like a step unlike the other points

Reply:
Good point, removed in 02278343335477482bab4f77ec7034adc481609e